lib/gameState.ex

```diff
@@ -0,0 +1,98 @@
+defmodule GameState do
+  def new_game_state do
+    %{
+      :available_moves => [1, 2, 3, 4, 5, 6, 7, 8, 9],
+      :win? => false,
+      :player_1_moves => [],
+      :player_2_moves => [],
+      :player_1_mark => "X",
+      :player_2_mark => "O",
+      :current_player => 1
+    }
+  end
```

Comment:
Some other ideas for naming could be:
-game
-game_data
-game_record
-data
-play_data

I'm keen to change it to game_data since that is what it is essentially keeping track of.